### DISCUSSION TITLE
Welcome to Hamster-App Discussions!

### DISCUSSION THREAD

**Discussion by WithSJ** [WithSJ/Hamster-App#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hasibulkarimratul** [comment]
How do you change screens ? ( I am extremely nooooob)

**Reply by Hasibulkarimratul** [reply]
Oh! you use Change Screen Function to change the screen. In there you used ScreenManager

**Reply by WithSJ** [reply]
define name to your screen and add that screen in ScreenManager after that you can switch screen using that name.

**Comment by Hasibulkarimratul** [comment]
Yeah it worked. But how to use a fucking callback function on any event :/
Tried stackoverflow solution,  docs. Both throws  error.

**Reply by WithSJ** [reply]
def func(self,obj=""):
    pass

in event callback
(callback=self.func)  don't use () after func we don't need to call that function here.
or 

you can use lambda function
callback= lambda x: self.func()

now i created another discussion for learning kivy and kivymd.
Q&A for learning on this : [click ->] https://github.com/WithSJ/Hamster-App/discussions/4

**Comment by Hasibulkarimratul** [comment]
Can I  get 'ur fb ID -,-? Might Knock you to get help

**Reply by WithSJ** [reply]
My Instagram : @with.sj
